### DISCUSSION TITLE
Password reset suggestion

### DISCUSSION THREAD

**Discussion by dscotese** [community/community#60272] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I hardly ever log into github from a machine on which I have never logged into github, but the login page says "email or username" and I usually (incorrectly) enter my email address.  This leads to a "bad password" error, and me requesting a password reset after being frustrated for a minute or so.  On the way to resetting my password, I remember that I'm probabkly not my email address, but rather my username.  When I start the password reset, and probably just when I submit the original login form, github could check to see if the email address is associated with an account that doesn't use the email address to log in, and mention that: "You entered the email address of a valid account, but that account has a username.  Try that instead!".

I suspect this would save a couple hundred people a few minutes every so often. :-)

**Comment by Joelfgndes** [comment]
Another issue is requesting to reset Github´s password and NEVER receiving the reset password EMAIL....no, also not in spam folder!

**Reply by LiteBrite82** [reply]
Thanks for posting in the GitHub Community, @dscotese !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Feed category is solely related to conversations around the GitHub product Feed. This question should be in the General category. I've gone ahead and moved it for you. Good luck!
